frontend/mock-api/concepts.json

```diff
@@ -138,6 +141,7 @@
             {
               "id": "budget",
               "label": "Budget",
+              "tooltip": "A money range tooltip, let's see how that works.",
```

Comment:
```suggestion
              "tooltip": "A money range tooltip, let's see how that works.\nMaybe it could benefit from newlines?",
```

Reply:
If not possible applicable, the PR can be merged

Reply:
Hehe, that won't work unfortunately

Reply:
What will work though is `Check this<br />newline`

Reply:
Changed it.